### PR TITLE
test: Adding fixture sample

### LINKED ISSUE #118: Add fixtures with test data

### PR DESCRIPTION
**Pull request by philter87**
## Description

- This PR adds a data sample: app/fixtures/sample.json. The data is added to the database with ```just add-test-data``` or ```poetry run python manage.pu loaddata sample```

## Related Issues

Closes #118 

## Testing

- [ ] Yes
- [X] No, not needed (give a reason below)
- [ ] No, I need help writing them

## Reviewer Focus

Understand the format in "sample.json", which adds a table and a column

## Checklist

For all PRs that are not general documentation

- [] Tests accompany or reflect changes to the code
- [X] Tests ran and passed locally
- [ ] Ran the linter and formatter
- [X] Build has passed locally
- [X] Relevant documentation has been updated

### CODE REVIEW

File: app/fixtures/sample.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  {
+    "model": "app.ColumnMetaData",
+    "pk": 1,
+    "fields": {
+      "table_metadata_id": 1,
+      "name": "text_column",
+      "title": "Text Column",
+      "description": "Description of column",
+      "data_type_id": 1,
```
Comment (signekb) [inline app/fixtures/sample.json:21]:
```suggestion
      "data_type": 1,
```
Not sure whether that's changed in a PR somewhere, but on `main` this field is called `data_type` not `data_type_id`

**Review by signekb — COMMENTED** [review]
Looks good! Just one comment on the naming of the `data_type` column in the `ColumnMetadata` model, before I can approve

File: app/fixtures/sample.json (hunk 1 above)
Comment (philter87) [reply app/fixtures/sample.json:21]: data_type becomes data_type_id in the database (it references a data_type by the id of the data_type). Hence, data_type_id is also valid

Comment (signekb) [reply app/fixtures/sample.json:21]: Ah, I see!

**Review by signekb — DISMISSED** [review]

**Comment by lwjohnst86** [comment]
I tried running this and it doesn't work. It says 

```
django.db.utils.OperationalError: Problem installing fixture '/home/luke/Documents/organizations/seedcase/seedcase-sprout/app/fixtures/sample.json': Could not load app.TableMetadata(pk=1): no such table: app_tablemetadata
```

**Comment by philter87** [comment]
> I tried running this and it doesn't work. It says
> 
> ```
> django.db.utils.OperationalError: Problem installing fixture '/home/luke/Documents/organizations/seedcase/seedcase-sprout/app/fixtures/sample.json': Could not load app.TableMetadata(pk=1): no such table: app_tablemetadata
> ```

Ahhh... The migrations are missing. I will add the migration step to the just command

**Review by K-Beicher — APPROVED** [review]
Looks like we discussed :-)

**Comment by signekb** [comment]
@philter87 When I run `just add-test-data`, I get an IntegrityError: 
```
django.db.utils.IntegrityError: Problem installing fixtures: The row in table 'app_columnmetadata' with primary key '1' has an invalid foreign key: app_columnmetadata.data_type_id contains a value '1' that does not have a corresponding value in app_columndatatype.id.
error: Recipe `add-test-data` failed on line 51 with exit code 1
```

**Review by signekb — APPROVED** [review]
👍

**Pull request merged**